paasta_tools/kubernetes_tools.py

```diff
@@ -3432,3 +3434,44 @@ def update_crds(
             success = False
 
     return success
+
+
+def get_kubernetes_secret_name(
+    service_name: str, secret_name: str, namespace: str = "paasta"
+) -> str:
+    service = sanitise_kubernetes_name(service_name)
+    sanitised_secret = sanitise_kubernetes_name(secret_name)
+    name = f"{namespace}-secret-{service}-{sanitised_secret}"
+    return name
+
+
+def get_kubernetes_secret(secret_name: str, service_name: str, cluster: str) -> str:
+    k8s_secret_name = get_kubernetes_secret_name(service_name, secret_name)
+    template_name = f"{{.data.{secret_name}}}"
+
+    env = os.environ.copy()
+    env["KUBECONFIG"] = DEFAULT_KUBECTL_CONFIG_PATH
+
+    proc = subprocess.run(
```

Comment:
can we the python kubernetes library instead of using subprocess?

Reply:
done.